examples/video-post.ts

```diff
@@ -0,0 +1,70 @@
+import { Wechaty } from '../src/wechaty'
+import { VideoPost } from '../src/mod'
+
+const bot = new Wechaty({
+  name : 'video-post-bot',
+})
+
+const videoPost = new VideoPost({
+  authorId: 'todo',
+  coverageUrl: 'todo',
+  title: 'todo',
+  videoUrl: 'todo',
+})
+
+async function testVideoPost () {
+
+  /**
+   * Video Post
+   */
+
+  // send video post
+  const message = bot.Message.load('')
```

Comment:
the `Message.load()` is not a public API. (as well as all user classes's `load()` is private)

Please remove this method from our example code and figure another way to build the message.

Thank you very much.

Reply:
Thanks for point out this problem.